### ISSUE TITLE
Suggestion: Statistics page which shows transfer rates and amounz

### ISSUE THREAD

**Issue by HT-7** [wolpi/prim-ftpd#213]
I suggest a statistics page which shows the current upload/download speeds and the totally download and uploaded data volume.

It would also be good if it displayed the list of recent file actions in real time.

**Comment by wolpi** [comment]
Yes, such statistics could be interesting. I fear I won't find time to look into this soon, but contributions are welcome :wink:
